docusaurus/README.md

```diff
@@ -0,0 +1,58 @@
+# Documentation and Docusaurus
+
+We use [docusaurus](https://docusaurus.io) for consistent process, in `Airbyte` **no website is generated**.
+Functionally this is a very fancy **linter**
+
+Running the build process will **check for broken links**, please read the output and address
+any broken links that you are able to do.
```

Comment:
Just wondering, is this something we rely on people who build the docs to check and fix, or do we have CI for it? I know we've had issues with broken links in the past

Reply:
CI will tell you!

Reply:
Sweet!

Reply:
There's a caveat: Docusaurus link checker only checks for internal links to markdown files. If there's a link to a website with a full URL, even our docs site, I don't know if it's going to be checked as a build step. I'll take a look.